src/FlightDisplay/GuidedActionsController.qml

```diff
@@ -217,7 +217,7 @@ Item {
     property var _actionData
 
     on_FlightModeChanged: {
-        _vehiclePaused =        activeVehicle ? _flightMode === activeVehicle.pauseFlightMode : false
+        _vehiclePaused =        activeVehicle ? _flightMode === activeVehicle.pauseFlightMode || activeVehicle.guidedMode: false
```

Comment:
I'm going to have to play around with this change a bit. In hindsight I don't quite remember why it also checked guided mode. There may be something important there.

Reply:
Sorry I had this backward. You added the guided mode check. Why? This causes pause to not show up in some cases where it should.